Userland/DevTools/HackStudio/LanguageClient.cpp

```diff
@@ -127,7 +127,8 @@ void ConnectionToServer::declarations_in_document(DeprecatedString const& filena
 
 void ConnectionToServer::todo_entries_in_document(DeprecatedString const& filename, Vector<CodeComprehension::TodoEntry> const& todo_entries)
 {
-    ToDoEntries::the().set_entries(filename, move(todo_entries));
+    // FIXME: This copies the todo_entries, but IPC does not allow us to take the Vector by value
+    ToDoEntries::the().set_entries(filename, todo_entries);
```

Comment:
I've used `const_cast` to work around this in the past, see Userland/Libraries/LibSQL/SQLClient.cpp.